qontract_api/qontract_api/integrations/quay_repos/schemas.py

```
@@ -0,0 +1,142 @@
+    orgs: list[QuayOrgConfig] = Field(
+        ..., description="List of Quay organizations to reconcile"
+    )
+    dry_run: bool = Field(
+        default=True,
+        description="If True, only calculate actions without executing. Default: True (safety first!)",
+    )
+
+
+
+
+class QuayRepoActionCreate(BaseModel, frozen=True):
```

Ruff (0.16.1)

[error] Too many blank lines (4)

Remove extraneous blank line(s)

(E303)